### DISCUSSION TITLE
Cluster downscale

### DISCUSSION THREAD

**Discussion by ShaiShalev87** [kube-green/kube-green#314] [Q&A]
I understand kube green is running as pods in the cluster.
Which means it requires at least one node running.

Is it possible to use it in order to downscale all the worker nodes? (For example in gcp)
For some shutdown period (night, weekends, etc) 

Thanks, 
Shai

**Comment by davidebianchi** [comment]
Hi @ShaiShalevSQream! 
You are right, since kube-green is an operator, it must run inside the cluster. This means that at least 1 node should be running. It is anyhow possible to downscale all other cluster nodes.
Maybe, if nodes are very large, it is possible to use affinity to set kube-green in a smaller one.
